src/science/landuse/landuse3.F90

```diff
@@ -3,8 +3,49 @@ MODULE landuse_variable
   IMPLICIT NONE
 
   SAVE
-
+  !*# Overview of `landuse_variable` module (landuse3.F90)
+  !
+  ! This MODULE applies land-use changes for CABLE.
+  !
+  ! This includes:
+  !
+  ! - plant functional type transitions
+  ! - wood harvest
+  ! - land management
+  !
+  !## Method
+  !
+  ! All variables in the restart file and the state variables will be
```

Comment:
This needs to be clarified since Tammas has some questions and I'm not entirely sure of the correct reply.